src/test/resources/features/release_version_checksums.feature

```diff
@@ -0,0 +1,130 @@
+Feature: Release Version with checksums
+
+  Background:
+    Given the Consumer has a valid Auth Token
+
+  Scenario: Release binary for Linux 64 bit Platform with checksums
+    Given the existing Default PLATFORM_SPECIFIC java Version is 8u121-zulu
+    And the Consumer for candidate java|jmc is making a request
```

Comment:
We should only have one candidate here. Both of these are platform specific candidates. How about Scala, Kotlin or Groovy instead.

Reply:
Switched to groovy (UNIVERSAL)